js/views/standalone_registration_view.js

```diff
@@ -50,14 +53,40 @@
       'change #code': 'onChangeCode',
       'click #register': 'registerWithoutMnemonic',
       'click #register-mnemonic': 'registerWithMnemonic',
+      'click #back-button': 'onBack',
+      'click #save-button': 'onSaveProfile',
       'change #mnemonic': 'onChangeMnemonic',
       'click #generate-mnemonic': 'onGenerateMnemonic',
       'change #mnemonic-display-language': 'onGenerateMnemonic',
       'click #copy-mnemonic': 'onCopyMnemonic',
       'click .section-toggle': 'toggleSection',
-      'keyup #password': 'onPasswordChange',
+      'keyup #password': 'onValidatePassword',
       'keyup #password-confirmation': 'onValidatePassword',
     },
+    async showPage(pageIndex) {
+      // eslint-disable-next-line func-names
+      this.$pages.each(function(index) {
+        if (index !== pageIndex) {
+          $(this).hide();
+        } else {
+          $(this).show();
+        }
+      });
+    },
+    async showRegisterPage() {
+      this.registrationParams = {};
+      this.showPage(0);
```

Comment:
Should we have an enum style object or something for these indexes?
(Argument against would be they are only used in `showXPage` functions anyway)

Reply:
Hmm, good point, i'm not sure if it's actually needed though unless we have more than 2 pages 🤔